app/views/new_user/dossiers/index.html.haml

```diff
@@ -5,7 +5,7 @@
 
 .dossiers-headers.sub-header
   .container
-    = link_to "Commencer une nouvelle démarche", demarches_url, class: "button secondary new-demarche"
+    -# = link_to "Commencer une nouvelle démarche", demarches_url, class: "button secondary new-demarche"
```

Comment:
Hello,

nous ne passons jamais de code en commentaire : quand nous n’avons plus besoin d’une ligne de code, nous la supprimons.

Reply:
Je prends la modif

Reply:
Ok, c'est noté @fredZen pour les commentaires.